graphs/possible_bipartition.py

```diff
@@ -1,12 +1,41 @@
-# Can be used for BFS
-from collections import deque 
+from collections import deque
+
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n * m) where n is the number of nodes/dogs 
```

Comment:
⏱ Time complexity will actually be O(n + m) because you traverse each node and each edge exactly once. O(n*m) would mean that you traverse all the edges in the entire graph for each node in the graph.